.github/workflows/pytest.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install .
+        pip install toml
```

Comment:
Out of curiosity, do you have a strategy for tracking dependencies in a centralized manner?

Reply:
Not really, I do have the core dependencies as part of the `setup.py`, but optional requirements such as `toml` will have to be installed by the user